### ISSUE TITLE
Fix tab order for datetime fields - display element not focusable

### ISSUE THREAD

**Issue by wildone** [typerefinery-ai/typerefinery-websight#410]
Tab order does not work for datetime fields. When display is visible, it should be focusable (tabindex=0). When input is visible, it should be in tab order. Need keyboard navigation support (Enter/Space to activate display).
